src/view.c

```diff
@@ -436,16 +435,18 @@ view_apply_tiled_geometry(struct view *view, struct output *output)
 }
 
 static void
-view_apply_fullscreen_geometry(struct view *view, struct wlr_output *wlr_output)
+view_apply_fullscreen_geometry(struct view *view)
 {
-	assert(wlr_output);
-	struct output *output =
-		output_from_wlr_output(view->server, wlr_output);
+	assert(view->fullscreen);
+	if (!is_enabled_output(view->output)) {
+		return;
```

Comment:
We may want to prevent ever calling `apply_fullscreen_geometry()` when the output is not available, e.g. we could use an `assert()` here instead. If the output for a fullscreen view is gone, I think we should simply disable the fullscreen and then let the usual logic to move views that are not within the layout anymore take over.

Reply:
We do have logic to do that (disable fullscreen) if needed in response to a layout change. However I'm struggling to convince myself that an `assert` here would never be hit. As a compromise, how about just logging an error before the `return`?

Reply:
Looking at it again, the `assert` is fine. Added.